en_US/data-integration/data-bridge-iotdb.md

```diff
@@ -0,0 +1,174 @@
+
+Now the Apache IoTDB data bridge should appear in the data bridge list (**Data Integration** -> **Data Bridge**) with Resource Status as Connected. 
+
+### Create a Rule for Apache IoTDB Data Bridge
+
+You can continue to create a rule to forward data to the new Apache IoTDB bridge.
+
+1. Go to the EMQX Dashboard, and click **Integration -> Rules**.
+
+2. Click **Create** on the top right corner of the page.
+
+3. Input a rule ID, for example, `my_rule`.
+
+4. Input the following statement in the SQL editor, which will forward the MQTT messages matching the topic pattern `t/#`:
+
+   ```
+   SELECT
+     *
+   FROM
+     "t/#"
+
+   ```
+
+5. Click the **Add Action** button, select **Forwarding with Data Bridge** from the dropdown list, and then select the data bridge you just created under **Data Bridge**.
+
+6. Click the **Add** button to finish the setup.
+
+7. Click the **Create** button at the bottom of the page to finish the setup.
+
+Now a rule to forward data to Apache IoTDB via the data bridge is created. You can click **Data Integration** -> **Flows** to view the topology. It can be seen that the messages under the topic `t/#` are sent and saved to Apache IoTDB.
+
+### Test the Data Bridge and Rule
+
+You can use the built-in WebSocket client in the EMQX dashboard to test your rule and bridge.
+
+1. Click **Diagnose** -> **WebSocket Client** in the left navigation menu of the Dashboard.
+
+2. Fill in the connection information for the current EMQX instance. 
+
+   - If you run EMQX locally, you can use the default value.
+   - If you have changed EMQX's default configuration. For example, the configuration change on authentication can require you to type in a username and password.
+
+3. Click **Connect** to connect the client to the EMQX instance.
+
+3. Scroll down to the publish area and type in the following:
+   * **Topic**:`t/test`
+   * **Payload**: Make sure to select `JSON` as the payload encoding and then enter:
+   ```json
+   {
+     "measurement": "temp",
+     "data_type": "FLOAT",
+     "value": "37.6",
+     "device_id": "root.sg27"
+   }
+   ```
+   * **QoS**: `2`
+   
+4. Click **Publish** to send the message.
+
+If the data bridge and rule are successfully created, the message should have been published to the specified time series table in the Apache IoTDB server. You can check this using IoTDB's command line interface. If you're using it from docker as above you can connect the server by using this command from your terminal:
+
+```shell
+    $ docker exec -ti iotdb-service /iotdb/sbin/start-cli.sh -h iotdb-service
+```
+
+Then, from within this console, type the following:
+```sql
+IoTDB> select * from root.sg27
```

Comment:
This did not work for me. The message was inserted in `root.t.hej`. My guess is that t.hej is coming from the MQTT topic which was `t/hej`.